test/robot.test.js

```diff
@@ -0,0 +1,21 @@
+const { expect } = require('chai');
+const { move } = require('../src/robot');
+const buildVillageMap = require('../src/villageMap');
```

Comment:
I am not sure this code will work based on the changes you did [here](https://github.com/barabashlyuda77/robot/pull/2/files#diff-137c33186a1c9e76cc3e6191d2ba3a2cR22) 

I would recommend to setup [travis ci](https://travis-ci.org). So that your tests will run on every commit. You can try to follow the documentation: https://docs.travis-ci.com/user/getting-started/ and feel free to ask for help when needed